ingest.py

```diff
@@ -71,7 +71,7 @@ def populate(self):
         '''
 
         rigs = [r for r in RigDataPath().fetch(as_dict=True)
-                if r['rig'].startswith('RRig')]  # todo?: rig 'type'? Change between TRig and RRig for now
+                if r['rig'].startswith('TRig')]  # todo?: rig 'type'? Change between TRig and RRig for now
```

Comment:
is this check actually still needed? My thinking was that we check all directories in sequence; the water restriction and date should be unique across rigs, and so the sequence of ingest of TRig before RRig would be preserved (or could be made explicit from an ```order by``` in the query)

Reply:
Also, the 'discover' populate was mis-matched w/r/t to this when I updated the batch behavior ingest, though this could reflect local files which are a mock-up of what you actually are using

Reply:
side-note, the README probably should be updated with expected filesystem heirarchy and some sort of simple documentation on the ingested ```.mat``` file format